concourse/pipelines/rhui-release.yaml

```diff
@@ -0,0 +1,175 @@
+        path: sh
+    task: deploy-cds
+- name: deploy-prod-us-central1
+  plan:
+  - get: cds-image
+    passed:
+    - wave-0-finished
+    trigger: true
+  - get: rhua-image
+    passed:
+    - wave-0-finished
+    trigger: true
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - -exc
+        - gcloud compute instance-groups managed rolling-action replace --quiet rhua-mig-prod-us-central1 --region=us-central1 --project=google.com:rhel-infra; gcloud compute instance-groups managed wait-until --version-target-reached --quiet rhua-mig-prod-us-central1 --region=us-central1 --project=google.com:rhel-infra;
+        path: sh
+    task: deploy-rhua
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - -exc
+        - gcloud compute instance-groups managed rolling-action replace --quiet cds-mig-prod-us-central1 --region=us-central1 --project=google.com:rhel-infra; gcloud compute instance-groups managed wait-until --version-target-reached --quiet cds-mig-prod-us-central1 --region=us-central1 --project=google.com:rhel-infra;
+        path: sh
+    task: deploy-cds
+- name: deploy-prod-asia-southeast1
+  plan:
+  - get: cds-image
+    passed:
+    - wave-0-finished
+    trigger: true
+  - get: rhua-image
+    passed:
+    - wave-0-finished
+    trigger: true
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - -exc
+        - gcloud compute instance-groups managed rolling-action replace --quiet rhua-mig-prod-asia-southeast1 --region=asia-southeast1 --project=google.com:rhel-infra; gcloud compute instance-groups managed wait-until --version-target-reached --quiet rhua-mig-prod-asia-southeast1 --region=asia-southeast1 --project=google.com:rhel-infra;
+        path: sh
+    task: deploy-rhua
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - -exc
+        - gcloud compute instance-groups managed rolling-action replace --quiet cds-mig-prod-asia-southeast1 --region=asia-southeast1 --project=google.com:rhel-infra; gcloud compute instance-groups managed wait-until --version-target-reached --quiet cds-mig-prod-asia-southeast1 --region=asia-southeast1 --project=google.com:rhel-infra;
+        path: sh
+    task: deploy-cds
+resource_types:
+- name: gce-img
+  source:
+    repository: gcr.io/gcp-guest/gce-img-resource
+  type: registry-image
```

Comment:
can we add the registry-image-private already so we don't get bitten later by forgetting it's needed? :)

Reply:
snippet to accomplish this:

```yaml
resource_types:
- name: registry-image-private
  type: registry-image
  source:
    repository: gcr.io/compute-image-tools/registry-image-forked
- name: gce-image
  type: registry-image-private
  source:
    google_auth: true
    repository: gcr.io/gcp-guest/gce-img-resource

resources:
- name: IMAGENAME
  type: gce-image
  source:
    project: PROJECT
```